tests/integration/utils.js

```diff
@@ -270,7 +270,7 @@ testUtils.ajax = PouchForCoverage.ajax;
 testUtils.uuid = pouchUtils.uuid;
 testUtils.parseUri = pouchUtils.parseUri;
 testUtils.errors = PouchForCoverage.Errors;
-testUtils.extend = require('js-extend').extend;
+testUtils.extend = pouchUtils.jsExtend;
```

Comment:
testUtils.extend = pouchUtils.extend;

because it's still exported as extend in packages/node_modules/pouchdb-for-coverage/src/utils.js  https://github.com/pouchdb/pouchdb/pull/5751/files#diff-42219049ecd23077146f457971885dc3R64